### ISSUE TITLE
Uncaught (in promise) TypeError: admob.BannerAd is not a constructor

### ISSUE THREAD

**Issue by azapater** [admob-plus/admob-plus#195]
I've been trying to migrate from admob-free to admob-plus for the last few days but I can't make it work. I've tried to use the basic example in the project in a browser (before implementing it in my App) but I can't pass the "interstitial.isLoaded()" promise. It always returns false. The plugin seems to be loaded correctly and I can access all the methods etc.  The console shows this info tho: 

2020-10-15 18:39:46.455 13123-13123/example.admob.basic I/chromium: [INFO:library_loader_hooks.cc(50)] Chromium logging enabled: level = 0, default verbosity = 0
2020-10-15 18:39:46.555 13123-13123/example.admob.basic I/cr_BrowserStartup: Initializing chromium process, singleProcess=false
2020-10-15 18:39:46.574 13123-13123/example.admob.basic E/chromium: [ERROR:filesystem_posix.cc(89)] stat /data/user/0/example.admob.basic/cache/WebView/Crashpad: No such file or directory (2)
2020-10-15 18:39:46.574 13123-13123/example.admob.basic E/chromium: [ERROR:filesystem_posix.cc(62)] mkdir /data/user/0/example.admob.basic/cache/WebView/Crashpad: No such file or directory (2)
2020-10-15 18:39:48.211 13123-13202/example.admob.basic I/chatty: uid=10197(example.admob.basic) Chrome_InProcGp identical 2 lines
2020-10-15 18:39:48.331 13123-13123/example.admob.basic I/chromium: [INFO:CONSOLE(56)] "Received Event: deviceready", source: file:///android_asset/www/js/index.js (56)
2020-10-15 18:39:49.190 13123-13123/example.admob.basic I/chromium: [INFO:CONSOLE(19)] "false", source: file:///android_asset/www/js/index.js (19)
2020-10-15 18:39:49.190 13123-13123/example.admob.basic I/chromium: [INFO:CONSOLE(25)] "showAds()", source: file:///android_asset/www/js/index.js (25)
2020-10-15 18:39:49.190 13123-13123/example.admob.basic I/chromium: [INFO:CONSOLE(26)] "Uncaught (in promise) TypeError: admob.BannerAd is not a constructor", source: file:///android_asset/www/js/index.js (26)

I could upload the project but I'm basically compiling the very exact same code of the basic example for Android but with my app id and banner ids in test mode.

I'm using Cordova 10 and compiling for SDK 29, so using android@9.0.0

**Comment by ratson** [comment]
@azapater `admob.BannerAd` stub is not added to browser API yet, but it works on Android and iOS.
I have updated the basic example.

**Comment by azapater** [comment]
@ratson Thanks for your reply.

I think I haven't explained myself properly. Those console messages I attached are from Android Studio after building and running the basic example and executing it straight in a Pixel 2 API 29. I'm not running Cordova on the browser. 

The console log with "false" is the result of the promise admob.interstitial.isLoaded()

Because the problem seems to be the line "new admob.BannerAd" I've also tried to use the showAdsOld() method I've spotted in the basic example, but nothing really happens. No errors in the log either. 

I attach a screenshot of the log straight from Android Studio. 

Thanks

![Screenshot_1](https://user-images.githubusercontent.com/1220658/96479903-7163de80-1231-11eb-86ba-26699f4f198c.png)
